File: cpp/59A.cpp
// 59 A. Word

#include <iostream>
#include <string>
using namespace std;

int main() {
    string s;
    cin >> s;
    int upper = 0, lower = 0;

    for (char c : s) {
        if (isupper(c)) upper++;
        else lower++;
    }

    if (upper > lower) {
        for (char &c : s) c = toupper(c);
    } else {
        for (char &c : s) c = tolower(c);
    }

    cout << s << endl;
    return 0;
}

File: java/59A.java
// 59 A. Word

import java.util.Scanner;

class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String s = sc.next();
        int upper = 0, lower = 0;

        for(char c : s.toCharArray()) {
            if(Character.isUpperCase(c)) upper++;
            else lower++;
        }

        if(upper > lower) {
            System.out.println(s.toUpperCase());
        } else {
            System.out.println(s.toLowerCase());
        }
        
    }
}